web_src/js/features/load-branches-and-tags.js

```diff
@@ -0,0 +1,53 @@
+const {csrfToken} = window.config;
+
+async function loadBranchesAndTags(loadingButton, addHere) {
+  loadingButton.setAttribute('disabled', 'disabled');
+  const response = await fetch(loadingButton.getAttribute('data-fetch-url'), {
+    method: 'GET',
+    headers: {'X-Csrf-Token': csrfToken},
+  }).finally(() => loadingButton.removeAttribute('disabled'));
+
+  if (!response.ok) {
+    return;
+  }
+
+  const data = await response.json();
```

Comment:
```suggestion
  let res;
  try {
    res = await fetch(loadingButton.getAttribute('data-fetch-url'), {
      method: 'GET',
      headers: {'X-Csrf-Token': csrfToken},
    });
  } finally {
    loadingButton.removeAttribute('disabled');
  }

  if (!res.ok) {
    return;
  }

  const data = await res.json();
```

I think I need to make a linter rule that enforces fetch responses to be named `res`, not `resp` and also not `response` 😆.

Reply:
But… _why_?
I mean, I agree that we need a common naming scheme but why `res` and not `resp` or `response`?`

Reply:
I guess it's an old habit of mine because in node.jq `res` (response) and `req` (request) are very common, see [this](https://nodejs.org/api/http.html).

Reply:
I really like `resp` and will continue using it.

`res` reads like `result`.